drizzlepac/alignimages.py

```diff
@@ -327,11 +329,18 @@ def run_align(input_list, archive=False, clobber=False, debug=False, update_hdr_
         for im in img:
             im.meta['name'] = image
         imglist.extend(img)
+    #store mapping of group_id to filename/chip
+    group_id_dict={}
+    for image in imglist:
+        group_id_dict["{}_{}".format(image.meta["filename"],image.meta["chip"])] = image.meta["group_id"]
 
     best_fit_rms = -99999.0
     best_fitStatusDict={}
     best_fitQual = 5
-    fit_algorithm_list= [match_2dhist_fit,match_default_fit]
+    # fit_algorithm_list = [match_default_fit,match_2dhist_fit,match_relative_fit] #TODO: UNCOMMENT before deployment
+    fit_algorithm_list = [match_default_fit,match_2dhist_fit] #TODO: REMOVE before deployment
+    orig_imglist = copy.deepcopy(imglist)
+    temp_imglist = []
```

Comment:
Please add a comment here as to the reason for the copy.

Reply:
Issue resolved by commit [77fbf75](https://github.com/spacetelescope/drizzlepac/pull/268/commits/77fbf75f4c69ebc263a37a92aa3a59e93f5848e4)